frontend/src/pages/LoginPage.js

```diff
@@ -80,67 +79,72 @@ class LoginPage extends Component{
 
     render() {
         return (
-            <div>
-                <div className="logo">
-                    <img
-                        src={`http://${process.env.REACT_APP_IP}/res/logo.png`}
-                        alt="Logo"
-                        width="340"
-                        onClick={this.onLogoClick}
-                    />
-                </div>
-                <div className="login-box-container">
-                    {this.state.redirect && <Redirect to={this.state.redirect} />}
-                    <p><font size ="5" color="#1DE28F"> Login </font></p>
-
-                    <form>
-                        <p>
+          <div>
+            <div className="logo">
+              <img
+                  src={`http://${process.env.REACT_APP_IP}/res/logo.png`}
+                  alt="Logo"
+                  onClick={this.onLogoClick}
```

Comment:
it was there before, I left it because I was unsure if the person who wrote it intended on linking something to the logo

Reply:
It was linked before, but it was since removed. The logo wont be linked to anything anymore, so please remove every occurrence of onLogoClick in both files.